source/slang/hlsl.meta.slang

```diff
@@ -9861,6 +9861,31 @@ uint GeometryIndex()
     }
 }
 
+// Get the vertex positions of the currently hit triangle in any-hit or closest-hit shader.
+// https://github.com/KhronosGroup/GLSL/blob/master/extensions/ext/GLSL_EXT_ray_tracing_position_fetch.txt
+__glsl_extension(GL_EXT_ray_tracing)
+__glsl_extension(GL_EXT_ray_tracing_position_fetch)
+__glsl_version(460)
+[ForceInline]
+float3 HitTriangleVertexPositions(uint index)
```

Comment:
Maybe name this `HitTriangleVertexPosition` since we are retrieving one position at a time.